vernac/vernacentries.ml

```diff
@@ -525,24 +526,23 @@ let post_check_evd ~udecl ~poly evd =
   else (* We fix the variables to ensure they won't be lowered to Set *)
     Evd.fix_undefined_variables evd
 
-let start_lemma_com ~program_mode ~poly ~scope ~kind ?hook thms =
+let start_lemma_com ~typing_flags ~program_mode ~poly ~scope ~kind ?hook thms =
   let env0 = Global.env () in
```

Comment:
We need to put the flags in the env here

Reply:
Indeed. That's an overkill if anyone really uses that :)

Reply:
Not sure what you mean by overkill. AFAICT the univ flag goes through the ugraph into the evar map, so without updating the env here it won't be used in the proof.